internal/services/nginx/nginx_deployment_resource.go

```diff
@@ -179,7 +189,37 @@ func (m DeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 			},
 		},
 
+		"scaling": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			ForceNew: false,
```

Comment:
we can omit `ForceNew: false,` here as the schema will assume this to be the case when `ForceNew` is not specified